cli/request_test.go

```diff
@@ -151,3 +152,71 @@ func Test_Request_ExtendWithContent(t *testing.T) {
 		}
 	}
 }
+
+func Test_Request_ParseGroupCLIargs(t *testing.T) {
+	type Expected struct {
+		Group    string
+		SliceIDs []string
+	}
+	testCases := []struct {
+		Input    []string
+		Expected Expected
+		Errorf   func(error) bool
+	}{
+		// Tests that on sliceIDs are returned when non where provided
+		{
+			Input: []string{"mygroup"},
+			Expected: Expected{
+				Group:    "mygroup",
+				SliceIDs: []string{},
+			},
+			Errorf: nil,
+		},
+		// Tests that group and slice are split correctly
+		{
+			Input: []string{"mygroup@1"},
+			Expected: Expected{
+				Group:    "mygroup",
+				SliceIDs: []string{"1"},
+			},
+			Errorf: nil,
+		},
+		// Tests that multiple group sliceIDs are split correctly
+		{
+			Input: []string{"mygroup@1", "mygroup@2"},
+			Expected: Expected{
+				Group:    "mygroup",
+				SliceIDs: []string{"1", "2"},
+			},
+			Errorf: nil,
+		},
+		// Tests that mixed groups return an invalidArgumentsError
```

Comment:
How about we split this into two test functions? One for success cases, one for failure cases? We don't need Expected for the latter, nor Errorf for the first.